libraries/shared/src/shared/Storage.h

```diff
@@ -14,6 +14,7 @@
 #include <vector>
 #include <memory>
 #include <functional>
+#include <stdexcept>
```

Comment:
Same here.

Reply:
I found that <stdexcept> was needed for compiling the project on Windows10 with Visual Studio 2019 and using the cmake that is included in VS. It obviously is not needed under Linux.